### ISSUE TITLE
Icehowl changing Charge direction

### ISSUE THREAD

**Issue by gugovgeorgi** [gamefreedomgit/Frostmourne#2581]
Description:
Icehowl should choose a Charge direction and keep it.
This works correctly, UNLESS the targeted player has a Hand Of Protection applied. 
When the HoP expires (or is removed in any way), Icehowl then changes his Charge direction to the point at which the 'targeted' player was when the HoP was removed
Clip: https://www.twitch.tv/iumruk/clip/AwkwardKawaiiRhinocerosNononoCat-9HuJ8rtaj0E1MRdC

It is clearly visible how he chooses to Charge to the side of the gate. It is visible that the target is a Shadow Priest with HoP applied (his DBM even shouts "Charge on me!")

Icehowl then procedes to charge in the middle of the gate,as the SP moved that way, and his HoP expired meantime.

How to reproduce:
Have Icehowl choose to Charge a player affected by HoP.
Let the player alter his position and remove the HoP, BEFORE the actual charge happens.
Icehowl will charge towards where the HoP was removed, not where he initially 'looked'

How it should work:
Icehowl always charges the direction he initially chose, i.e., 'looked' to.

**Comment by NickZedd** [comment]
Yeah he definitely changed direction, but I don't think he does it when the HoP runs out. The charge direction is probably determined when he jumps to the back wall or at some point after. He charged at the location where Swogs was standing at approx. 11 seconds into the clip, but you can see the HoP icon above his head for another second after that (it goes away when he jumps)

**Comment by NickZedd** [comment]
You should be able to play around this by not moving until everyone else stops being stunned I think?

**Comment by Riztazz** [comment]
Fixed, will close once it's live
